ui/components/app/edit-gas-fee-popover/network-status/network-status.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { COLORS } from '../../../../helpers/constants/design-system';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import I18nValue from '../../../ui/i18n-value';
+import Typography from '../../../ui/typography/typography';
+
+import StatusSlider from './status-slider';
+
+const NetworkStatus = () => {
+  const { gasFeeEstimates } = useGasFeeContext();
+  let estBaseFee = 0;
+  if (gasFeeEstimates?.estimatedBaseFee) {
+    // estimatedBaseFee is not likely to be below 1, value .01 is used as test networks sometimes
+    // show have small values for it and more decimal places may cause UI to look broken.
+    estBaseFee = parseFloat(gasFeeEstimates?.estimatedBaseFee);
```

Comment:
I think we have to be careful here, because ether can't be represented properly using floats (for instance, 15 WEI is `15 * 10**18`, which gets corrupted into `1.5000000000000002e-16`). I think the best thing here might be to run this through BigNumber. I feel like you might have run across this code already but I think you can do that by using `toBigNumber.dec()` from `shared/modules/conversion.utils` (unless there's another way you know to do this). Of course if this is already a BigNumber then you're good to go! But once you have a BigNumber you can call `toFixed(2)` on that to format it (while keeping it as a BigNumber).